freedomofpress/securedrop-client
# Update dev dependencies

# Description

Update dev dependencies

# Test Plan

0. Create a new virtual env via `make venv` and activate it
1. Run `make check`
2. - [ ] Confirm that it passes without error
3. - [ ] Confirm CI passes without error
4. - [ ] Confirm only dev depenedency files were updated

# Checklist

If these changes modify code paths involving cryptography, the opening of files in VMs or network (via the RPC service) traffic, Qubes testing in the staging environment is required. For fine tuning of the graphical user interface, testing in any environment in Qubes is required. Please check as applicable:


If these changes add or remove files other than client code, the AppArmor profile may need to be updated. Please check as applicable:


If these changes modify the database schema, you should include a database migration. Please check as applicable:
